brave/brave-core
# Storybook: Restore Wallet Page and Create Password Error Handling

Resolves <https://github.com/brave/brave-browser/issues/16203> and <https://github.com/brave/brave-browser/issues/16026>

## Submitter Checklist:

1) Built out the Restore Wallet Page
2) Added Error Handling to Create Password Forms



## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. `npm run storybook`
2. Confirm that storybook shows the new component.
3. Check functionality of the new component.

https://user-images.githubusercontent.com/40611140/120519733-319cd200-c390-11eb-84a3-d22aea50c2f3.mov
